add 'Teleport - City Spells' to masterlist

jb29 opened the pull request:
Hi! Here's the first of the three teleport spell mods from TheWulfPanda: [Teleport - City Spells](https://www.nexusmods.com/skyrimspecialedition/mods/7015/)

I didn't have the motivation to write a user friendly cleaning guide for the wild edits and upload it somewhere, so I just added a comment with the dirty records. I still hope that the author will read my message eventually and upload the cleaned ESPs.

Thanks to @CptMcSplody's suggestion I now checked the Bash Tags with the [Generate Tags for Wrye Bash.pas](https://github.com/fireundubh/xedit-scripts/blob/master/all/Generate%20Tags%20for%20Wrye%20Bash.pas) script. It suggested **C.Regions** but I omitted the tag because those changes are not relevant for the mod.

MacSplody commented on masterlist.yaml:
```diff
@@ -2895,6 +2895,53 @@ plugins:
+        condition: 'active("TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp")'
+        content: >
+          It seems that you also activated "TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp".
+          You should use only one of those two ESPs.
```
You could use the inc: key to achieve the same result as this message.

```
  - name: 'TheWulfPanda - City Teleport Spells.esp'
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/7015/'
        name: 'Teleport - City Spells on Nexus Mods'
    inc:
      - 'TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp'
```

MacSplody commented on masterlist.yaml:
```diff
@@ -2921,25 +2923,7 @@ plugins:
       #   - 00037EE9
       - crc: 0xB63A3D6B
         util: SSEEdit v3.2.1
-  - name: 'TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp'
```
It's fine to keep the plug-ins in separate entries. 
The above inc: will only work for `TheWulfPanda - City Teleport Spells.esp`
and the dirty plug-in info will not show for `TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp`

jb29 replied:
Ah okay, I thought I could squash it together into a single list entry. 😄 
I'm going to rebase my PR.

jb29 committed:
add 'Teleport - City Spells' to masterlist

MacSplody commented on masterlist.yaml:
```diff
@@ -2895,6 +2895,45 @@ plugins:
+      #   - 00042249
+      #   - 00020EE7
+      #   - 0001A27F
+      #   - 00037EE9
```
Also you don't have to link to a guide for the wild edits. You can do something like below. F4169312 is the CRC for `TheWulfPanda - City Teleport Spells.esp` with the wild edits.

```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - 'Remove the following records with xEdit: Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D],SolitudeOrigin [CELL:00037EE9],WhiterunOrigin [CELL:0001A27F],MarkarthOrigin [CELL:00020EE7],RiftenCityNorth [CELL:00042249]'
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

MacSplody replied:
So you can duplicate that message in the other entry and replace the esp name and CRC.
`TheWulfPanda - City Teleport Spells Less Magicka Increase Cast Time.esp  DE73AD5C`

jb29 replied:
Does the LOOT messaging system support line breaks or will they be ignored?
If we can use line breaks then I think something like this would be a bit more readable in the code as well as in the frontend:

```yaml
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |
            Remove the following records with xEdit:
              - Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
              - SolitudeOrigin [CELL:00037EE9]
              - WhiterunOrigin [CELL:0001A27F]
              - MarkarthOrigin [CELL:00020EE7]
              - RiftenCityNorth [CELL:00042249]
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

MacSplody replied:
~~I don't think it does, you would still need to wrap the string in Apostrophes ' '~~

Strings in Quotes " " can have new lines added with \n. 

![image](https://user-images.githubusercontent.com/1944639/42119481-6106c446-7c04-11e8-840d-0d8deae0c820.png)

```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - "\n\nRemove the following records with xEdit:\n- Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]\n- SolitudeOrigin [CELL:00037EE9]\n- WhiterunOrigin [CELL:0001A27F]\n- MarkarthOrigin [CELL:00020EE7]\n- RiftenCityNorth [CELL:00042249]"
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

jb29 replied:
YAML supports two types of multi-line strings, the pipe `|` and the greater than sign `>`.

Using `>` is actually the same as wrapping the string in quotes, but you can use line breaks to make the code more readable. The line breaks will be ignored and replaced with a whitespace.

**single-line:**
```yaml
my_message: 'line 1 line 2 line 3'
```

**pseudo multi-line:**
```yaml
my_message: >
  line 1
  line 2

  line 3
```

Both versions above will have the same **output**:
> line 1 line 2 line 3

But the `|` will actually pass the line breaks to the application.

**real multi-line:**
```yaml
my_message: |
  line 1
  line 2

  line 3
```
**output:**
> line 1
> line 2
>
> line 3

But to be able to render those multi-line strings, the application has to translate them into valid HTML.

I just looked into the [source code](https://github.com/loot/loot/blob/dev/src/gui/html/elements/loot-plugin-card.js#L230) and LOOT even supports [Markdown](https://guides.github.com/features/mastering-markdown/) to format the messages. So the example in my last comment would look something like this in LOOT:

> Remove the following records with xEdit:
>   - Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
>   - SolitudeOrigin [CELL:00037EE9]
>   - WhiterunOrigin [CELL:0001A27F]
>   - MarkarthOrigin [CELL:00020EE7]
>   - RiftenCityNorth [CELL:00042249]

MacSplody replied:
Okay it will work, it has been used in the past for common entries in the skyrim masterlist. 

```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |

            > 
            Remove the following records with xEdit:
            ```
              Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
              SolitudeOrigin [CELL:00037EE9]
              WhiterunOrigin [CELL:0001A27F]
              MarkarthOrigin [CELL:00020EE7]
              RiftenCityNorth [CELL:00042249]
            ```
```
Which would get you:
![image](https://user-images.githubusercontent.com/1944639/42122937-89b8688a-7c41-11e8-8c60-90f59cebdd5f.png)

HTML looks useful.
```
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |
            <p>
              <h3>Remove the following records with xEdit:</h3>
                <ul>
                  <li>Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]</li>
                  <li>SolitudeOrigin [CELL:00037EE9]</li>
                  <li>WhiterunOrigin [CELL:0001A27F]</li>
                  <li>MarkarthOrigin [CELL:00020EE7]</li>
                  <li>RiftenCityNorth [CELL:00042249]</li>
                </ul>
            </p>
```
![image](https://user-images.githubusercontent.com/1944639/42122975-d18a9bce-7c41-11e8-9c4b-26fab4a7d948.png)

@WrinklyNinja Is the use of  `|` and `>` okay?

Ortham replied:
Yes, you can use `|` and `>` to make the masterlist more readable, and `|` is preferable to using `\n`. I'd avoid using HTML as Markdown covers most text formatting options and is more readable in source form.

MacSplody replied:
@WrinklyNinja Okay so the below block that @jb29 suggested is fine format wise?

```yaml
    msg:
      - <<: *hasWildEdits
        subs:
          - 'TheWulfPanda - City Teleport Spells.esp'
          - |
            Remove the following records with xEdit:
              - Parent Sub-Block 2,-2 of MorthalExterior04 [CELL:0000939D]
              - SolitudeOrigin [CELL:00037EE9]
              - WhiterunOrigin [CELL:0001A27F]
              - MarkarthOrigin [CELL:00020EE7]
              - RiftenCityNorth [CELL:00042249]
        condition: 'checksum("TheWulfPanda - City Teleport Spells.esp",F4169312)'
```

Ortham replied:
Yes, that's fine.

MacSplody committed:
Changes for Pull request

- Add block for wild edit guide using `|`, the + is to maintain the newline before guide header.
- Add clean info for plug-ins with remaining wild edits.
- Change key order for dirty.
- Remove comments which are no longer required.

MacSplody commented:
@jb29 Well done and thanks for the contribution.
